content/en/tracing/profiling/proxy.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Proxy configuration with Profiling
+kind: Documentation
+further_reading:
+    - link: 'https://www.datadoghq.com/blog/introducing-datadog-profiling/'
+      tags: 'Blog'
+      text: 'Introducing always-on production profiling in Datadog.'
+---
+
+Profiles collected by the Datadog librairies are sent directly from your running application to Datadog. If your network configuration is restricted for outbound traffic use your proxy to redirect the profiles to Datadog.
```

Comment:
Profiles collected by the Datadog librairies -> Profiles collected by Datadog librairies